backend/app/main.py

```
@@ -33,6 +34,223 @@
+                        import os
+                        import subprocess
+
+                        # Get the alembic.ini path
+                        backend_dir = os.path.dirname(
+                            os.path.dirname(os.path.abspath(__file__))
+                        )
+
+                        logger.info("Executing Alembic upgrade to head...")
+
+                        # Run Alembic as subprocess to avoid output buffering issues
+                        result = subprocess.run(
```

Ruff (0.14.8)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)